Build Error: missing fields `SCL` and `SDA` in initializer of `CodecPins` at calling new_daisy_board!() on bin crate

`new_daisy_board!()` is for stealing peripherals so that users does not accidentally use them contrary to what Daisy board intends to.
So it should be used on the user side.
But using it on the `bin` crate that has `daisy_embassy` for its `dependencies` makes an error:
```
error[E0063]: missing fields `SCL` and `SDA` in initializer of `CodecPins`
  --> src/main.rs:36:17
   |
36 |     let board = new_daisy_board!(p);
   |                 ^^^^^^^^^^^^^^^^^^^ missing `SCL` and `SDA`
   |
```
This error is because there are no features called `seed_1_1` **on the bin crate**. [see where the actual error happens](https://github.com/Dicklessgreat/daisy_embassy/blob/4afc330f1b3a9ac45e4e11badca5d10527c8f559/src/lib.rs#L138)

Could someone suggest a solution or workaround?

Note: This error doesn't happens when you build examples of `daisy_embassy` on its repository.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -92,6 +92,36 @@ pub fn default_rcc() -> hal::Config {
     config
 }
 
+#[cfg(feature = "seed_1_1")]
+#[macro_export]
+macro_rules! codec_pins {
+    ($p:ident) => {
+        daisy_embassy::CodecPins {
+            // For audio, I2C only needed for WM8731
+            SCL: $p.PH4,
+            SDA: $p.PB11,
+
+            MCLK_A: $p.PE2,
+            SCK_A: $p.PE5,
+            FS_A: $p.PE4,
+            SD_A: $p.PE6,
+            SD_B: $p.PE3,
+        }
+    };
+}
+#[cfg(feature = "seed_1_2")]
+#[macro_export]
+macro_rules! codec_pins {
+    ($p:ident) => {
+        daisy_embassy::CodecPins {
+            MCLK_A: $p.PE2,
+            SCK_A: $p.PE5,
+            FS_A: $p.PE4,
+            SD_A: $p.PE6,
+            SD_B: $p.PE3,
+        }
+    };
+}
 #[macro_export]
 macro_rules! new_daisy_board {
     ($p:ident) => {
@@ -133,19 +163,7 @@ macro_rules! new_daisy_board {
 
             audio_peripherals: daisy_embassy::audio::AudioPeripherals {
                 codec: daisy_embassy::Codec {},
-                codec_pins: daisy_embassy::CodecPins {
-                    // For audio, I2C only needed for WM8731
-                    #[cfg(feature = "seed_1_1")]
-                    SCL: $p.PH4,
-                    #[cfg(feature = "seed_1_1")]
-                    SDA: $p.PB11,
-
-                    MCLK_A: $p.PE2,
-                    SCK_A: $p.PE5,
-                    FS_A: $p.PE4,
-                    SD_A: $p.PE6,
-                    SD_B: $p.PE3,
-                },
+                codec_pins: daisy_embassy::codec_pins!($p),
                 sai1: $p.SAI1,
                 i2c2: $p.I2C2,
                 dma1_ch0: $p.DMA1_CH0,

